docs/repo-specific-configuration.md

```diff
@@ -3,6 +3,34 @@
 You can add `<YOUR_REPO>/.scala-steward.conf` to configure how Scala Steward updates your repository.
 
 ```properties
+# pullRequests.frequency allows to control how often or when Scala Steward
+# is allowed to create pull requests.
+#
+# Possible values:
+#   @asap
+#     PRs are created without delay.
+#
+#   @daily | @weekly | @monthly
+#     PRs are created at least 1 day | 7 days | 30 days after the last PR.
+#
+#   <CRON expression>
+#     PRs are created roughly according to the given CRON expression.
+#
+#     CRON expressions consist of six fields: seconds, minutes, hour of day,
+#     day of month, month, and day of week.
+#
+#     See https://www.alonsodomin.me/cron4s/userguide/index.html#parsing for
+#     more information about the CRON expressions that are supported.
```

Comment:
maybe add a link to web based cron tan generator like https://crontab.guru/

Reply:
I'd love to but cron4s requires 6 fields and the generator has only 5.

Reply:
you are right. I missed that.

Reply:
I'm thinking out loud: we'll never be able to take the "seconds" field into account, so maybe we should reduce the number of fields to 5 by always prepending `* ` to the CRON expressions. That way we'll only the support the more common format that uses only 5 fields.

Reply:
I like this idea. 👍 
As the crontab syntax is cryptic and we have no possibility to interact with the user. Making it simpler is a good idea.